`pthread.h` include triggers compilation error

```c
#include <pthread.h>

int main(int argc, char *argv[])
{
  return 0;
}
```

```shell
$ flatpak run --command=bash --filesystem=home:rw org.freedesktop.Sdk//20.08
[📦 org.freedesktop.Sdk ~]$ rm -f /tmp/foo; /usr/bin/clang  -o /tmp/foo ~/tmp/test.c
[📦 org.freedesktop.Sdk ~]$ rm -f /tmp/foo; /usr/lib/sdk/llvm11/bin/clang  -o /tmp/foo ~/tmp/test.c
In file included from /var/home/phil/tmp/test.c:1:
In file included from /usr/include/pthread.h:21:
/usr/include/features.h:465:12: fatal error: 'sys/cdefs.h' file not found
#  include <sys/cdefs.h>
           ^~~~~~~~~~~~~
1 error generated.
[📦 org.freedesktop.Sdk ~]$ cat /run/flatpak/ld.so.conf.d/runtime-003-org.freedesktop.Sdk.Extension.llvm11.conf 
/usr/lib/sdk/llvm11/lib
```

```shell
[📦 org.freedesktop.Sdk ~]$ rm -f /tmp/foo; /usr/lib/sdk/llvm11/bin/clang -v -o /tmp/foo ~/tmp/test.c
clang version 11.1.0
Target: x86_64
Thread model: posix
InstalledDir: /usr/lib/sdk/llvm11/bin
 "/usr/lib/sdk/llvm11/bin/clang-11" -cc1 -triple x86_64 -emit-obj -mrelax-all -disable-free -disable-llvm-verifier -discard-value-names -main-file-name test.c -mrelocation-model static -mframe-pointer=all -fmath-errno -fno-rounding-math -mconstructor-aliases -munwind-tables -target-cpu x86-64 -fno-split-dwarf-inlining -debugger-tuning=gdb -v -resource-dir /usr/lib/sdk/llvm11/lib/clang/11.1.0 -fdebug-compilation-dir /var/home/phil -ferror-limit 19 -fgnuc-version=4.2.1 -fcolor-diagnostics -faddrsig -o /tmp/test-75a5ef.o -x c /var/home/phil/tmp/test.c
clang -cc1 version 11.1.0 based upon LLVM 11.1.0 default target x86_64
ignoring nonexistent directory "/usr/local/include"
#include "..." search starts here:
#include <...> search starts here:
 /usr/lib/sdk/llvm11/lib/clang/11.1.0/include
 /usr/include
End of search list.
In file included from /var/home/phil/tmp/test.c:1:
In file included from /usr/include/pthread.h:21:
/usr/include/features.h:465:12: fatal error: 'sys/cdefs.h' file not found
#  include <sys/cdefs.h>
           ^~~~~~~~~~~~~
1 error generated.
```

--- a/org.freedesktop.Sdk.Extension.llvm11.json
+++ b/org.freedesktop.Sdk.Extension.llvm11.json
@@ -36,14 +36,14 @@
           "aarch64": {
             "config-opts": [
               "-DFFI_INCLUDE_DIR=/usr/lib/aarch64-linux-gnu/libffi-3.2.1/include",
-              "-DLLVM_DEFAULT_TARGET_TRIPLE=aarch64",
+              "-DLLVM_DEFAULT_TARGET_TRIPLE=aarch64-unknown-linux-gnu",
               "-DLLVM_TARGETS_TO_BUILD='AArch64'"
             ]
           },
           "x86_64": {
             "config-opts": [
               "-DFFI_INCLUDE_DIR=/usr/lib/x86_64-linux-gnu/libffi-3.2.1/include",
-              "-DLLVM_DEFAULT_TARGET_TRIPLE=x86_64",
+              "-DLLVM_DEFAULT_TARGET_TRIPLE=x86_64-unknown-linux-gnu",
               "-DLLVM_TARGETS_TO_BUILD='X86;AMDGPU;NVPTX'"
             ]
           }

